packages/server/src/cli.ts

```diff
@@ -83,10 +97,6 @@ const bold = (text: string | number): string | number => {
 		readonly extraArgs?: string;
 	};
 
-	if (options.disableTelemetry) {
```

Comment:
If the user launches code-server with `--disable-telemetry`, the telemetry won't be disabled without this block. This should be reverted.